core/models/ResourcesTable.ts

```diff
@@ -304,7 +304,7 @@ export interface IResourcesRecord {
   frameId: number;
   socketId: number;
   protocol: string;
-  type: ResourceType;
+  type: IResourceType;
```

Comment:
Should this be the enum? Are we just trying to reduce the need for a dev to have to import and use the enum?